### PR TITLE
Capstone 1 - Chelmerrox Photography website

### PR DESCRIPTION
**Pull request by chelmerrox**
Hi! This PR is for the Capstone 1 project. I've completed its requirements to:

- follow the design guidelines by Cindy Shin on Behance
- personalize my website
- create the mobile & desktop versions of the Home/main page & the About page
- create a responsive mobile menu when the hamburger icon is tapped in both pages
- dynamically create the Featured work sections on the Home/main page using JavaScript
- link the website logo to the home page 
- link the About link in Home page to About page

Thanks in advance for your review, comments & help!

<a href="https://www.loom.com/share/f70f99e9c9b74be790ab7e49c87690da">Link to my video presentation</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        </div>
+      </div>
+
+      <!--Section 2 | Photography Niche for mobile screens-->
+      <section class="section-2-mobile row d-flex d-sm-flex d-md-none bg-darkgrey px-2 py-5">
+        <h2 class="heading lato-font text-center fw-bold col-12 col-sm-12">Photography Niche</h2>
+
+        <div class="section-2-1-mobile col-12 col-sm-12 container-fluid mt-5">
+          <!--Nature-->
+          <div class="row section-2-1-1-mobile mb-3 py-3">
```
Comment (Kossi-stack) [inline index.html:115]: - [x] Kindly give the same size to your image so that all fit well.
Capturecap

File: index.html [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+            <img src="images/243346.svg" alt="Fiji Water logo" class="fiji-water-logo col-4 col-sm-4"/>
+          </div>
+        </div>
+      </section>
+
+      <!--Section 4 | Partners for tablet & desktop screens-->
+      <section class="section-4-desktop row d-none d-sm-none d-md-flex px-2 py-5 bg-darkgrey">
+        <h2 class="heading lato-font font-darkgrey text-white text-center fw-bold col-md-12">Partners</h2>
+
+        <div class="section-4-1-desktop col-md-12 d-md-flex flex-md-row justify-content-ms-around mt-5 ps-5">
```
Comment (Kossi-stack) [inline index.html:225]: - [x] Kindly fix the image.
Capturecf

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @chelmerrox,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Ismailco) [inline index.html:115]: - [x] As the previous reviewer mentioned, kindly give all the cards the same (size) width so your design looks more e profissional

Your design looks like this for now 
Screenshot_20220622_231113

File: index.html [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+        <div class="section-1-3-desktop col-md-12 d-md-flex flex-md-column align-items-md-start">
+          <h4 class="cocogoose-font font-darkgrey fs-3" id="smartphone-photography-text">SMARTPHONE PHOTOGRAPHY</h4>
+          <h6 class="description lato-font font-darkgrey fw-bold fs-5">of nature, art, culture & many more!</h6>
+        </div>
+      </div>
+
+      <!--Section 2 | Photography Niche for mobile screens-->
+      <section class="section-2-mobile row d-flex d-sm-flex d-md-none bg-darkgrey px-2 py-5">
+        <h2 class="heading lato-font text-center fw-bold col-12 col-sm-12">Photography Niche</h2>
+
```
Comment (Ismailco) [inline index.html:112]: - [x] [OPTIONAL] Kindly consider containing the headers of the `Photography Niche`, `Featured Work` and `Partners` with the orange `hr` so your design looks more apeling and aligns properly

Screenshot_20220622_231801

Screenshot_20220622_231734

Screenshot_20220622_231748

File: styles/style.css [hunk 4]
```diff
@@ -0,0 +1,1072 @@
...
+  box-sizing: border-box;       /* box-sizing is a property allows us to change exactly how the box-model works  */
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (Ismailco) [inline styles/style.css:16]: - [x] [OPTIONAL] You can reduce the font size on your page so your design looks like the provided one

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @chelmerrox,

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
## Hi @chelmerrox 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**